libcaf_test/caf/test/runnable.hpp

```diff
@@ -265,6 +265,10 @@ class CAF_TEST_EXPORT runnable {
       return unbox_all(vals, std::index_sequence_for<Ts...>{});
   }
 
+  size_t block_parameters_index() const {
```

Comment:
This function is missing comment for documentation.
```suggestion
  /// Returns the `index` of block_parameter for the running test.
  size_t block_parameters_index() const {
```